files/bootstrap.sh

```diff
@@ -322,7 +322,7 @@ if [[ "$MACHINE" != "x86_64" && "$MACHINE" != "aarch64" ]]; then
 fi
 
 if [ "$MOUNT_BPF_FS" = "true" ]; then
-  sudo mount-bpf-fs
+  mount-bpf-fs
```

Comment:
`sudo` isn't necessary here because the bootstrap script runs as `root`. Removing it allows `mount-bpf-fs` to be mocked in the unit tests.